README.md

```diff
@@ -39,6 +39,30 @@ Note that Pulp operator works with three different types of service containers (
 
 <br>Pulp operator is manually built and [hosted on quay.io](https://quay.io/repository/pulp/pulp-operator). Read more about the container images [here](https://docs.pulpproject.org/pulp_operator/container/).
 
+## Custom Resource Definitions
+Pulp Operator currently provides 3 different kinds of [Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#custom-resources): Pulp, Pulp Backup and Pulp Restore.
+### Pulp
+Manages pulp application and its deployments, services, ..., through the following ansible roles:
```

Comment:
```suggestion
Manages the pulp application and its deployments, services, ..., through the following ansible roles:
```

Reply:
This is mostly personal preference but I'd go with capitalized Pulp everywhere :)